src/elements/content-sidebar/versions/__tests__/VersionsSidebarContainer.test.js

```diff
@@ -32,6 +32,7 @@ describe('elements/content-sidebar/versions/VersionsSidebarContainer', () => {
     const api = {
         getVersionsAPI: () => versionsAPI,
     };
+
```

Comment:
Revert? Do we have any tests for the `onLoad` call?

Reply:
I will likely add one to to the code in EUA? unless you think its a good idea to have one here too.

Reply:
We generally want to have unit tests on both sides to cover the logic for each individual component.

Reply:
ack, will add one here too :)